src/pybind/mgr/cephadm/services/nfs.py

```diff
@@ -141,7 +173,7 @@ def remove_rgw_keyring(self, daemon: DaemonDescription) -> None:
         entity: AuthEntity = self.get_auth_entity(f'{daemon_id}-rgw')
 
         logger.info(f'Removing key for {entity}')
-        ret, out, err = self.mgr.check_mon_command({
+        ret, out, err = self.mgr.mon_command({
```

Comment:
Let it be `check_mon_command`, it checks the return value. We can remove `ret, out, err`.